src/assets/i18n/it/translation.json

```
@@ -0,0 +1,97 @@
+  "standardised-name": "Cognome Standardised",
+  "date-of-birth": "Data di nascita",
+  "sex": "Sesso",
+  "male": "Maschio",
+  "female": "Femmina",
+  "other": "Altro",
+  "unknown": "Sconosciuto/a",
+  "day": "Giorno",
+  "month": "Mese",
+  "year": "Anno",
+  "address": "Indirizzo",
+  "zip": "Codice postale",
+  "city": "Città",
+  "street": "Via",
+  "house-number": "Numero Civico",
+  "phone-number": "Numero di Telefono",
+  "email-address": "E-Mail",
+  "test-id": "Test-ID*",
+  "processing-consent": "Consenti il processo nel CWA",
+  "bill-status": "stato della fattura",
+  "data-privacy-approve": "Approva il trattamento dei dati",
+  "patientdata-exclude": "Aggiungi i dati del paziente nel QR-Code",
+  "patient-data-correction": "Correggi i dati del paziente",
+  "process-finish": "Finisci il processo",
+  "qr-code": "QR Code",
+  "patient-data": "Dati del paziente",
+  "cancel": "Cancella",
+  "data-submit": "Inserisci i dati",
+  "result-positiv": "Positivo",
+  "result-negative": "Negativo",
+  "result-failed": "Fallito",
+  "process-number": "Pratica numero",
+  "logout": "Disconnetti",
+  "wrong-process-number": "Pratica numero errata",
+  "unknown-process-number": "Pratica numero {{processNo}}, sconosciuta",
+  "server-error": "Errore del sistema: {{status}}",
+  "server-not-reachable": "Il sistema non è raggiungibile",
+  "connection-error": "Errore di connessione: {{message}}",
+  "qr-scan": "Scansiona il QR-Code",
+  "qr-scan-https-only": "QR-Scan funziona solo con il protocollo https",
+  "qr-code-no-patient-data": "Il codice QR-code non contiene dati del paziente",
+  "error-message": "Messaggio di errore",
+  "serverError": "É accaduto un errore:",
+  "error-patient-data-load": "Non siamo riusciti a caricare i dati del paziente.",
+  "statistics": "Statistiche",
+  "failed-report": "Impossibile caricare la pratica per il test",
+  "totalTestCount": "Test totali",
+  "positiveTestCount": "Test Positivi Totali",
+  "successfull-transferred": "Dati caricati correttamente",
+  "record-vaccination-cert-dat": "Certificazione della Pratica di Vaccinazione",
+  "vaccination-cert": "Certificato di Vaccinazione",
+  "identifierType": "Tipo di Identificazione",
+  "country": "Nazionalitá del paziente",
+  "identifierNumber": "Numero identificativo",
+  "disease-agent": "Malattia / Agente",
+  "def-disease-agent": "Malattia o Agente interessato (e.g. 840539006)",
+  "vaccine": "Vaccino/Prophylaxis",
+  "def-vaccine": "Descrizione generica del vaccino/prophylaxis o dei suoi componenti (e.g. 1119305005)",
+  "vac-medical-product": "Prodotto medico",
+  "def-vac-medical-product": "Codice del medicinale (e.g. EU/1/20/1528)",
+  "vac-marketing-holder": "Sistema di gestione delle organizzazioni",
+  "def-vac-marketing-holder": "Codice definito nell'EMA SPOR - Sistema di gestione delle organizzazioni (e.g. ORG-100030215)",
+  "sequence": "Numero di dose",
+  "def-sequence": "Numero di dosi administrate in un ciclo (e.g. 2)",
```

```suggestion
  "def-sequence": "Numero di dosi somministrate in un ciclo (e.g. 2)",
```

"Administrate" is not an italian word. We use "somministrare" that becomes "somministrate" in this context.